admin/js/links-tab.js

```diff
@@ -385,6 +385,17 @@ jQuery(function ($) {
             config_table.find('tr.supports_create input[type=checkbox]').prop('checked', checked);
           }
 
+          if (type_obj['meta']['supports_logo'] ) {
+            config_table.find('tr.display_logo').show();
+            let checked = false;
+            let link_obj = fetch_link_obj($('#ml_main_col_available_link_objs_select').val());
+            if (link_obj && link_obj['type_config'] && link_obj['type_config']['display_logo']) {
+              checked = true;
+              has_config = true;
+            }
+            config_table.find('tr.display_logo input[type=checkbox]').prop('checked', checked);
+          }
+
```

Comment:
@cairocoder01 I'm also doing some work in and around this area; so you take the lead with regards to merging your latest logic first and I'll rebase and tweak accordingly...

Reply:
@kodinkat I'm not sure what you mean. I see no other changes in master that would need to be merged in to this.

Reply:
I'm also implementing a connections enabled config; which I'll aim to push up a little later... To avoid any workflow merge conflicts, I'm happy to have your work merged into master first and then I'll rebase accordingly, so as to ensure I've not broken anything! 😉

Reply:
...having said that, I could rebase directly onto your branch, but; just in case it's still subject to change, maybe not...

Reply:
I don't plan on this being subject to change unless there's specific comments on this PR. You're welcome to just merge the PR and then work off of it. 😉